webpack/assets/javascripts/react_app/components/SearchBar/__tests__/__snapshots__/SearchBar.test.js.snap

```diff
@@ -34,7 +34,7 @@ exports[`AutoComplete rendering renders AutoComplete 1`] = `
         Object {
           "canCreate": true,
           "controller": "models",
-          "documentationUrl": "http://www.theforeman.org/manuals/1.19/index.html#4.1.5Searching",
+          "documentationUrl": "/doc/url",
```

Comment:
is this on purpose?

Reply:
yes, to remove versioning from snapshots